CopilotKit/packages/backend/src/lib/openai-assistant-adapter.ts

```diff
@@ -177,6 +184,77 @@ export class OpenAIAssistantAdapter implements CopilotKitServiceAdapter {
       };
     }
   }
+
+  async buildInstructions(forwardMessages: Message[]): Promise<string> {
```

Comment:
the existing `submitUserMessage` function already performs the instruction-building and tool-building steps clearly enough, there is no need for defining these as separate functions